ethereum/datastructures/src/main/java/tech/pegasys/teku/datastructures/attestation/ValidateableAttestation.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.datastructures.attestation;
+
+import com.google.common.base.Objects;
+import com.google.common.primitives.UnsignedLong;
+import java.util.Collection;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicBoolean;
+import org.apache.tuweni.bytes.Bytes32;
+import tech.pegasys.teku.datastructures.operations.Attestation;
+import tech.pegasys.teku.datastructures.operations.IndexedAttestation;
+import tech.pegasys.teku.datastructures.operations.SignedAggregateAndProof;
+
+public class ValidateableAttestation {
+  private final Attestation attestation;
+  private final Optional<SignedAggregateAndProof> maybeAggregate;
+  private final AtomicBoolean gossiped;
```

Comment:
Initial value is always false so probably should just use
```suggestion
  private final AtomicBoolean gossiped = new AtomicBoolean(false);
```

Reply:
Makes sense. Done.